appium/api-mocks/apis/ekm/ekm-endpoints.ts

```diff
@@ -62,298 +62,422 @@ const throwErrorIfConfigSaysSo = (config: EkmConfig) => {
   }
 }
 
+// pub, primaryFingerprint, name, date are used to check Keys in KeyScreen and can be optional
+export interface KeyDetailInfo {
+  prv: string;
+  pub?: string;
+  primaryFingerprint?: string;
+  name?: string;
+  date?: string;
```

Comment:
What does the date represent here?

Reply:
This is to check if key screen displays keys correctly.
we can't get name, date from appium ui tests so I added those infos in key details.
let me know your thought.

![image](https://user-images.githubusercontent.com/77344191/165712408-3c0744bc-4797-4302-b8ac-5ae39f8aaf2e.png)

Reply:
Let's call them `renderedPrimaryUid`, and then check in the app what does the date represent. Is it date created or date last modified? Based on that, call it `renderedDateCreated` or `renderedDateLastModified`

Reply:
`renderedPrimaryUid` doesn't seem to correct naming as it contains name + email

Reply:
like `Test1 <test1@example.net>`

Reply:
That is exactly how user id (uid) is encoded in the public/private key. It's correct :-)

Reply:
ok :-)